pkg/network/config/config_test.go

```diff
@@ -135,25 +135,14 @@ func TestEnableHTTPMonitoring(t *testing.T) {
 	})
 }
 
-func TestEnableKafkaMonitoring(t *testing.T) {
+func TestEnableDataStreams(t *testing.T) {
```

Comment:
you should still check the environment variable

Reply:
That didn't work, tried it